CONTRIBUTING.md

```diff
@@ -0,0 +1,38 @@
+## How to contribute to software.opensuse.org
+
+#### **Did you find a bug?**
+
+* **Do not open up a GitHub issue if the bug is a security vulnerability
+  in the application**, and instead contact the [security team](mailto:security-team@suse.de).
+
+* **Ensure the bug was not already reported** by searching on GitHub under [Issues](https://github.com/openSUSE/software-o-o/issues).
+
+* If you're unable to find an open issue addressing the problem, [open a new one](https://github.com/openSUSE/software-o-o/issues/new). Be sure to include a **title and clear description**, as much relevant information as possible, including screenshots of what you see in your browser. You can attach screenshots to the text by drag and dropping them.
+
+#### **Did you write a patch that fixes a bug?**
+
+* Open a new GitHub pull request with the patch.
+
+* Ensure the PR description clearly describes the problem and solution. Include the relevant issue number if applicable.
```

Comment:
We should add that screenshots are appreciated for esthetic changes. I can add it in another PR if you don't want to change it :wink: